releasenotes/notes/exporter-proxy-serializer-fix-1d9751fdd91955f0.yaml

```diff
@@ -0,0 +1,3 @@
+fixes:
```

Comment:
Could you add another release note for the change to support `datadogexporter.proxy_url`?